### ISSUE TITLE
bmenu -u and bad url

### ISSUE THREAD

**Issue by ychaouche** [HadrienAka/TheBashMenu#1]
Hello Hadrien,

bmenu -u looks for https://hadrienaka.fr/update.txt which is a redirect to https://dns.google/update.txt, I'm not sure that was inteded. 
https://hadrienaka.fr isn't responding either.

![image](https://user-images.githubusercontent.com/1773693/132376231-b205c952-c36b-4590-91f5-b50beae8ca47.png)

**Comment by HadrienAka** [comment]
Yep it's "normal", I don't have anymore the property of hadrienaka.fr
I'm searching a new way to do it
